sonic_package_manager/manifest.py

```diff
@@ -213,6 +213,8 @@ def unmarshal(self, value):
         ManifestRoot('container', [
             ManifestField('privileged', DefaultMarshaller(bool), False),
             ManifestArray('volumes', DefaultMarshaller(str)),
+            ManifestField('network', DefaultMarshaller(str), ''),
+            ManifestArray('ports', DefaultMarshaller(str)),
```

Comment:
Could you please add new fields (with usage examples preferably) to https://github.com/sonic-net/SONiC/blob/master/doc/sonic-application-extension/sonic-application-extension-guide.md?